### PR TITLE
ETC Ecosystem Spotlight: HebeSwap Part I

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is part I of a two part series that will explain what is HebeBlock and HebeSwap. It explains some of their products, the HEBE token, several other things the team does, and leaves for part II the guide of how to trade on HebeSwap.

"ETC Ecosystem Spotlight: HebeSwap Part I"

EN.

CN done.

This post will be published on June 20th 2023. Please approve.

### CODE REVIEW

File: content/blog/2023-06-20-etc-ecosystem-spotlight-hebeswap-part-I/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Staking HEBE, ETC, and Other Coins
+
+![Staking on liquidity pools.](./5.png)
+
+As to providing liquidity to the HebeSwap system of ERC-20 token trading pairs, users may purchase HEBE and match it with ETC, USDT, or other tokens to add volume to the corresponding LPs.
+
+They call this “Staking” or “liquidity mining” as an analogy to earning passive income from providing capital in proof of stake chains or from mining ETC.
+
+The way it works, is that users may deposit cryptocurrencies into the liquidity pools in the proportions of the market price for each pair. Then, when traders pay commissions when they buy and sell these pairs, part of the commissions goes to pay interest to the liquidity providers.
```
Comment (TheCrowbill) [inline content/blog/2023-06-20-etc-ecosystem-spotlight-hebeswap-part-I/index.md:79]: Line 79: Remove unneeded comma; Reword for readability
```suggestion
The way it works is that users may deposit cryptocurrencies into the liquidity pools in the proportions of the market price for each pair. Then, when traders pay commissions as they buy and sell these pairs, part of the commissions is paid as interest to the liquidity providers.
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by TheCrowbill — APPROVED** [review]
LG 👍

**Review by brolal — APPROVED** [review]

File: content/blog/2023-06-20-etc-ecosystem-spotlight-hebeswap-part-I/index.md
Comment (gitr0n1n) [inline content/blog/2023-06-20-etc-ecosystem-spotlight-hebeswap-part-I/index.md:1]: "it is growing organically and is a fundamental piece of infrastructure for the ecosystem"

- This product ran an ICO for 6,000 ETC if I remember correctly. This was not organic growth.

Comment (gitr0n1n) [reply content/blog/2023-06-20-etc-ecosystem-spotlight-hebeswap-part-I/index.md:1]: "The HebeBlock team projects that they will be launching a DAO so the community may vote on development, upgrades, and compensation issues.

They estimate that when the token rises in value as the ecosystem grows, that the monthly issuance may be significantly reduced. For this they will have the DAO setup for the community to vote on these issues."
- @DonaldMcIntyre as you know, this isn't good lingo to have around as the SEC is hammering ICO's for being unregistered securities extracting funding from US participants. Especially as CEX's are delisting projects with these liabilities attached to them.

Also on the point of violating regulations and the legal liabilities that follow a project; let's also not forget Hebe's "gateway.hebeswap.com" which was a non-kyc centralized bridge that opened their team up to FinCEN regulation violations. Outside of it being a very dumb idea of taking in stablecoin form the public, buying BUSD on Binance, then calling it USDT on Ethereum Classic's network.

**Review by gitr0n1n — CHANGES_REQUESTED** [review]
Please address comments before merging.

**Comment by gitr0n1n** [comment]
@ethereumclassic/website might want to make sure @DonaldMcIntyre doesn't go rouge like we've seen with other ETC Coop employees (stev).

**Comment by gitr0n1n** [comment]
@ethereumclassic/website  Note, 2 weeks later this bridge is materially compromised again. https://twitter.com/multichainorg/status/1677096839731097600

image


>"All the external tokens, meaning the tokens that are not issued inside ETC but exist in other chains, are bridged into the Ethereum Classic network through a technology called Multichain."

>Please note as the team that organized the Multichain bridge (Ethereum Classic DAO/EthereumClassic.com project/ETCswap). We advised people to pull their liquidity from that bridging due to centralization risk back in November 2022. In May 2023, their CEO went missing and the product appears to be in trouble given the lack of KYC and exposed centralized custody of user assets. This is why over $200K of liquidity left ETCswap in November 2022. I would view Multichain bridge as "deprecated" at this moment and it seems unwise you promote users putting more liqudity into that cross-chain routing protocol. @IstoraMandiri and I are on calls talking about this when our team organized that bridge. Then I've been on numerous community calls alerting to that centralized risk of losing your bridged assets.

https://github.com/ethereumclassic/ethereumclassic.github.io/pull/1252#discussion_r1234312508

**Comment by gitr0n1n** [comment]
Well it happened ~20 days after this warning. The bridged assets are stuck. @ethereumclassic/website you allowed Donald to throw a tantrum and get his way at the expense of the ETC community. The warning was clear as day and instead he used social engineering and exploited his roles in this repo to push this content despite valid and provable concerns on the content of this post. A good point to reflect on how this was handled.

image

https://twitter.com/MultichainOrg/status/1677180114227056641

**Pull request merged**